src/System.Security.Cryptography.X509Certificates/tests/CtorTests.cs

```diff
@@ -236,10 +239,16 @@ public static void TestCopyConstructor_Lifetime_Cloned()
             TestPrivateKey(c2, true);
 
             c1.Dispose();
+            GC.Collect();
+            GC.WaitForPendingFinalizers();
```

Comment:
You mean leaving in the calls to Collect/WaitForPendingFinalizer to try to force finalizers to run?  I'm ok with it.